pull_request_package/lib/obs_pull_request_package.rb

```diff
@@ -21,7 +21,15 @@ def url
     "https://build.opensuse.org/package/show/#{obs_project_name}/obs-server"
   end
   
+  def last_commited_sha
```

Comment:
Do you think we also should check the sha in the GitHubStatusReporter? Could it happen that the build result does not match with the sha of the PR?

Reply:
Not with packages with _service - because we first update the package and wait for the service to trigger. Then it's 'blocked' and the build result should match.

The alternative is a little hairy as we'd need to fetch jobhistory and compare with the sourcemd5.